src/tools/cleaner.js

```diff
@@ -43,11 +43,35 @@ const recurseReplaceBank = (obj, bank = {}) => {
     if (typeof out !== 'string') {
       return out;
     }
+
     Object.keys(bank).forEach(key => {
       // Escape characters (ex. {{foo}} => \\{\\{foo\\}\\} )
-      const s = String(key).replace(/[-[\]/{}()\\*+?.^$|]/g, '\\$&');
-      const re = new RegExp(s, 'g');
-      out = out.replace(re, bank[key]);
+      const escapedKey = String(key).replace(/[-[\]/{}()\\*+?.^$|]/g, '\\$&');
+      const matchesKey = new RegExp(escapedKey, 'g');
+
+      if (!matchesKey.test(out)) {
+        return;
+      }
+
+      const valueParts = out.split(/({{.*?}})/).filter(Boolean);
+      const replacementValue = bank[key];
+      const isPartOfString = valueParts.length > 1;
+
+      if (isPartOfString) {
+        if (_.isArray(replacementValue) || _.isPlainObject(replacementValue)) {
```

Comment:
I think here we can use native `Array.isArray` instead of lodash's `_.isArray`.

Reply:
Totally agree. Only used `_.isArray` since we were grabbing `_.isPlainObject` right next to it. No strong feeling about it though